config/clusters/pangeo-hubs/coessing.values.yaml

```diff
@@ -0,0 +1,43 @@
+basehub:
+  userServiceAccount:
+    annotations:
+      # iam.gke.io/gcp-service-account: pangeo-hubs-prod@pangeo-integration-te-3eea.iam.gserviceaccount.com
```

Comment:
Do you need to replace this annotation?

Reply:
Yep! Just done, and I had to run terraform too!